package.json

```
@@ -48,8 +48,10 @@
     "test:env:esm": "yarn --cwd tests/env/esm && yarn --cwd tests/env/esm start",
     "test:env:nitro-app": "yarn build && yarn --cwd tests/env/nitro-app test",
     "size": "node scripts/file-size ./dist/bundles/meilisearch.esm.min.js ./dist/bundles/meilisearch.umd.min.js",
-    "style": "yarn lint",
-    "style:fix": "yarn lint:fix",
+    "style": "yarn fmt && yarn lint",
+    "style:fix": "yarn fmt:fix && yarn lint:fix",
+    "fmt": "prettier -c ./**/*.{js,ts,tsx}",
+    "fmt:fix": "prettier -w ./**/*.{js,ts,tsx}",
```

I would just ask to remove this `,tsx` because it will prevent running the command later:
`[error] No files matching the pattern were found: "./**/*.tsx".`

```suggestion
    "fmt": "prettier -c ./**/*.{js,ts}",
    "fmt:fix": "prettier -w ./**/*.{js,ts}",
```

Unless you have a way to fix it without removing!